frontend/public/components/modals/_modals.scss

```diff
@@ -96,6 +111,10 @@
   background-color: var(--pf-global--BackgroundColor--100);
   margin-top: 0;
   padding: var(--pf-global--spacer--md) var(--pf-global--spacer--xl) var(--pf-global--spacer--xl) var(--pf-global--spacer--xl);
+
+  .alert {
```

Comment:
We should have already removed all `alert`s, so this should not be necessary.

Reply:
removed